### ISSUE TITLE
comments not supported in config file

### ISSUE THREAD

**Issue by spuriousbugfix** [yongkangchen/remote-sync#104]
.remote-sync.json file containing  /* comments */ results in:
SyntaxError: Unexpected token /

**Comment by nickv2002** [comment]
Strictly speaking, comments are not supported in JSON, so this doesn't sound like a bug.

**Comment by spuriousbugfix** [comment]
hmm... 
https://plus.google.com/+DouglasCrockfordEsq/posts/RK8qyGVaGSr

not a bug, but makes keeping the config management easier.

**Comment by nickv2002** [comment]
I agree that having comments would make config management easier here.
